.github/workflows/ci_metrics.yml

```diff
@@ -0,0 +1,24 @@
+name: CI Metrics
+
+on:
+  workflow_run:
+    workflows:
+      - "end-to-end"
+      - "Linting"
+      - "Tests"
+      - "REST API Tests"
+    types:
+      - completed
+  pull_request:
+    types:
+      - opened
+      - closed
```

Comment:
Just curious about these triggers, I imagined the `workflow_run` ones would be the only ones needed

Reply:
It gathers different metrics depedending on the triggering event, that's why I added also `pull_request`.
The action's README is more detailed.
https://github.com/int128/datadog-actions-metrics